docs/UserGuide.md

```diff
@@ -458,7 +458,7 @@ Format: `add -sl s/STUDENT_INDEX... l/LESSON_INDEX...`
 Examples:
 * `add -sl s/1 2 3 l/1 2` add students with index 1, 2, 3 into lessons with index 1, 2.
 
-> :exclamation: All of these students must not be attending any of the lessons provided for this command to work.
+> :bulb: Lessons have capacities - and the command won't work if any of the lessons cannot accommodate all the students.
 
 ### Deleting students from lessons: `del -sl`
```

Comment:
Maybe it is better to change `Deleting students from lessons` to `Deleting student(s) from lesson(s)` to standardise with `add -sl`? If so, need to update the command summary table as well